app/models/dish.rb

```diff
@@ -16,6 +16,8 @@ class Dish < ActiveRecord::Base
   scope :by_date, -> { order('created_at') }
   scope :by_name, -> { order('name') }
 
+  MAX_DEBT = 80
```

Comment:
Are you sure the basic currency unit in this case is a `złoty` and not a `grosz`? Lot of payment systems operate on the smallest currency units (like cents etc).

Reply:
Lunchiatto operates on `Money` instances when it comes to currency management and I believe it takes the care of the issue.